pkg/kube/snapshot/snapshot_test.go

```diff
@@ -403,7 +403,10 @@ func (s *SnapshotTestSuite) testVolumeSnapshot(c *C, snapshotter snapshot.Snapsh
 
 	snapshotName := snapshotNamePrefix + strconv.Itoa(int(time.Now().UnixNano()))
 	wait := true
-	err = snapshotter.Create(ctx, snapshotName, s.sourceNamespace, pvc.Name, snapshotClass, wait)
+	label := map[string]string{
+		"snapshottest": "testlabel",
+	}
+	err = snapshotter.Create(ctx, snapshotName, s.sourceNamespace, pvc.Name, snapshotClass, wait, label)
```

Comment:
Please add a test where Create is called with a label map that is not nil and with no keys in it. That will help clarify the behavior of List when such a map is provided ( I am guessing it will return nothing ? ) .

Reply:
No it will create it without any labels. List creates a matcher out of the given labels.